geo/src/algorithm/concave_hull.rs

```diff
@@ -0,0 +1,432 @@
+use crate::algorithm::convexhull::ConvexHull;
+use crate::algorithm::euclidean_distance::EuclideanDistance;
+use crate::algorithm::euclidean_length::EuclideanLength;
+use crate::prelude::Centroid;
+use crate::utils::partial_min;
+use crate::{Line, LineString, MultiLineString, MultiPoint, MultiPolygon, Point, Polygon};
+use num_traits::Float;
+use rstar::{RTree, RTreeNum};
+use std::collections::VecDeque;
+
+pub trait ConcaveHull<T> {
+    /// Returns a polygon which covers a geometry. Unlike convex hulls, which also cover
```

Comment:
I noticed that these doc comments are better visible if they are on the trait itself, rather than on the method.  Given there's only one method, it makes sense to move the comments to just above the trait declaration.